### PR TITLE
Removing IE and legacy edge

### PR DESCRIPTION
**Pull request by sammeboy635**
Cleaning up IE and Legacy Edge from Wptagent. 

Tested Chrome, Firefox, and Chromium Edge

## Chromium Edge
Microsoft

## Chrome
Chrome

## Firefox
firefox

@pmeenan Please Review!

### CODE REVIEW

**Review by pmeenan — CHANGES_REQUESTED** [review]
Looks good except for the browser detction where it should also add the detected Chromium-based edge as "Microsoft Edge" and "Edge"

File: wptagent.py [hunk 1]
```diff
@@ -745,44 +736,6 @@ def find_browsers(options):
...
-            logging.exception('Error getting windows build, skipping check for legacy Edge')
-        if edge is not None:
-            edge['type'] = 'Edge'
-            if 'Microsoft Edge (EdgeHTML)' not in browsers:
-                browsers['Microsoft Edge (EdgeHTML)'] = dict(edge)
-            if 'Microsoft Edge' not in browsers:
-                browsers['Microsoft Edge'] = dict(edge)
-            if 'Edge' not in browsers:
-                browsers['Edge'] = dict(edge)
         # Microsoft Edge (Chromium)
```
[the remark this answers is not in the document]
Comment (sammeboy635) [reply wptagent.py:739]: `
        # Microsoft Edge (Chromium)

        paths = [program_files, program_files_x86, local_appdata]

        channels = ['Microsoft Edge','Edge', 'Edge Dev']
        for channel in channels:`
Adding them to channels?

[the remark this answers is not in the document]
Comment (sammeboy635) [reply wptagent.py:739]: Thanks for clarifying! I stepped through the code on windows just to make sure it was correct with the adjustments we made and this is what browsers{} gets populated with after. If you would like to see more tests or output please do let me know!
image

File: wptagent.py [hunk 2]
```diff
@@ -797,20 +750,19 @@ def find_browsers(options):
                             browsers['Edgium'] = {'exe': edge_path}
                         elif channel == 'Edge Dev' and 'Edgium Dev' not in browsers:
                             browsers['Edgium Dev'] = {'exe': edge_path}
+                    browser_name = 'Microsoft {0}'.format(channel)
+                    if browser_name not in browsers:
+                        browsers['Microsoft Edge'] = {'exe': edge_path}
```
Comment (pmeenan) [inline wptagent.py:755]: This should be `browsers[browser_name] = `

File: wptagent.py [hunk 3]
```diff
@@ -797,20 +750,19 @@ def find_browsers(options):
...
+                    browser_name = '{0}'.format(channel)
+                    if browser_name not in browsers:
+                        browsers['Edge'] = {'exe': edge_path}
```
Comment (pmeenan) [inline wptagent.py:758]: Same. This should be `browsers[browser_name] = `

**Review by pmeenan — CHANGES_REQUESTED** [review]
One last fix and then I think we're there (beta wouldn't populate correctly without the fix).

**Review by pmeenan — APPROVED** [review]
LGTM.  @tkadlec - before merging, ping me for a few contacts you may want to ping before removing the legacy support as they may be using it in private instances.

**Pull request closed without merging**